README.md

```diff
@@ -2,13 +2,28 @@
 
 [Delight](https://www.datamechanics.co/delight) is a free and cross-platform Spark UI replacement with new metrics and visualizations that will delight you!
 
-## Timeline
+## Overview
 
-- June 2020: Project starts with a [blog post](https://www.datamechanics.co/blog-post/building-a-better-spark-ui-data-mechanics-delight) detailing our vision.
-- November 2020: First release consisting of a free hosted Spark History Server (no new metrics and visualizations yet).
-- Q1 2021 (expected): Release of the overview screen with new metrics and visualizations.
+The [Delight web dashboard](https://delight.datamechanics.co) lists your completed Spark applications with high-level information and metrics.
+<p align="center">
+<a href="documentation/images/delight_dashboard.png"><img src="documentation/images/delight_dashboard.png" width="80%" align="middle"></a>
+</p>
 
-We hope that the current release (free hosted Spark History Server) will be useful to the Spark developers who cannot easily spin up a SHS on their Spark platform.
+When you click on a specific line, you can view Executor CPU Metrics, aligned with a timeline of your Spark jobs and stages, so that it's easy for you to understand what is the performance bottleneck of your application. 
```

Comment:
nit: what the performance bottleneck of your application is (this `is` is at the end)

Reply:
thanks, will fix